scripts/quests/sandoria/The_Trader_in_the_Forest.lua

```diff
@@ -140,19 +140,18 @@ quest.sections =
                         return quest:progressEvent(127, xi.items.CLUMP_OF_BATAGREENS)
                     end
                 end,
+            },
+            onEventFinish =
```

Comment:
Yep, absolutely a syntax issue with the current implementation.  Could you please put a newline before the onEventFinish declaration?

Reply:
Done

Reply:
Awesome, thanks!  Just one more CI issue and this PR will be good to go:
![image](https://user-images.githubusercontent.com/81713309/175836990-ccabda7c-9242-4f8c-b0b9-872dbfbb2e3e.png)

Reply:
Should be fixed.